src/sentry/issues/endpoints/project_events.py

```diff
@@ -31,26 +38,42 @@ class ProjectEventsEndpoint(ProjectEndpoint):
         }
     }
 
+    @extend_schema(
+        operation_id="List a Project's Error Events",
+        parameters=[
+            GlobalParams.ORG_ID_OR_SLUG,
+            GlobalParams.PROJECT_ID_OR_SLUG,
+            CursorQueryParam,
+            OpenApiParameter(
+                name="full",
+                description="If this is set to true then the event payload will include the full event body, including the stacktrace. Set to 1 to enable.",
+                required=False,
+                type=bool,
+                location="query",
+                default=False,
+            ),
+            OpenApiParameter(
+                name="sample",
+                description="Return events in pseudo-random order. This is deterministic, same query will return the same events in the same order.",
```

Comment:
nit: slightly better wording?
```suggestion
                description="Return events in pseudo-random order. This is deterministic so an identical query will always return the same events in the same order.",
```

Reply:
This is better!